cmd/sqlflowserver/main.go

```diff
@@ -81,6 +85,7 @@ func main() {
 	caCrt := flag.String("ca-crt", "", "CA certificate file.")
 	caKey := flag.String("ca-key", "", "CA private key file.")
 	port := flag.Int("port", 50051, "TCP port to listen on.")
+	isArgoMode := flag.Bool("argo-mode", false, "Enable Argo workflow model.")
```

Comment:
maybe also need to configure the k8s API endpoint address in the future?

Reply:
Good point and I will add it when implementing the `Fetch` interface.